examples/plot_run_benchmark.py

```diff
@@ -13,7 +13,8 @@
 from benchopt.plotting import plot_benchmark, PLOT_KINDS
 
 
-BENCHMARK_PATH = Path(os.getcwd()).parent / 'benchmarks' / 'logreg_l2'
+BENCHMARK_PATH = Path(os.getcwd()).parent / \
+    'benchmarks' / 'benchmark_logreg_l2'
```

Comment:
Default is the current dir, no need for `os` here
```suggestion
BENCHMARK_PATH = (
    Path().parent / 'benchmarks' / 'benchmark_logreg_l2'
)
```